wasm/tools/fixtures/generation.py

```diff
@@ -23,7 +23,7 @@ def generate_fixture_tests(metafunc: Any,
     - `fixtures_dir` is the base path that fixture files will be collected from.
     """
     if 'fixture_path' in metafunc.fixturenames:
-        all_fixture_paths = tuple(sorted(find_json_fixture_files(fixtures_dir)))[:58]
+        all_fixture_paths = tuple(sorted(find_json_fixture_files(fixtures_dir)))
```

Comment:
Was pleasantly surprised that when I realized this mistake that there was only a single unicode tests which still needed to be cleaned up.